src/ngx_http_lua_kong_macos_arm64.h

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright 2019-2022 Kong Inc.
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+
+ *    http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#ifndef _NGX_HTTP_LUA_KONG_MACOS_ARM64_H_INCLUDED_
+#define _NGX_HTTP_LUA_KONG_MACOS_ARM64_H_INCLUDED_
+
+
+#include <ngx_config.h>
+#include <ngx_core.h>
+#include <ngx_http.h>
+
+
+/* macOS with Apple Silicon fixes, see: https://github.com/LuaJIT/LuaJIT/issues/205 */
```

Comment:
```suggestion
/*
 * macOS with Apple Silicon fixes,
 * see: https://github.com/LuaJIT/LuaJIT/issues/205
 */
```

Just minor Nginx style correction.